openexchangerates/src/config.ts

```diff
@@ -0,0 +1,6 @@
+import { Requester } from '@chainlink/external-adapter'
+import { Config } from '@chainlink/types'
+
+export const DEFAULT_ENDPOINT = 'latest.json'
```

Comment:
As with the other EA I think this should be changed to be used as other `endpoint`s are.